ResearchKit/Common/ORKTaskViewController.m

```diff
@@ -995,6 +1001,41 @@ - (void)observedScrollViewDidScroll:(UIScrollView *)scrollView {
     self.hairline.alpha = alpha;
 }
 
+- (NSArray *)stepsForReviewStep:(ORKReviewStep *)reviewStep {
+    NSMutableArray *steps = [[NSMutableArray alloc] init];
+    if (reviewStep.isStandalone) {
+        for (ORKStep *step in reviewStep.steps) {
+            [steps addObject:step];
+        }
+    } else {
+        ORKStep *nextStep = reviewStep;
```

Comment:
Say a study has lots of steps, but they don't necessarily need to be taken one after another. It would be useful if a participant could get an overview of what steps to do. He can then pick one, jump right in and go back to the overview when finished. I need this exact use case in my own app :-)

It basically is a review step that is shown before it's steps.